### ISSUE TITLE
After unlocking the language on the A7M4, select the language near Arabic.  Camera restarts automatically

### ISSUE THREAD

**Issue by hammer1987** [ma1co/Sony-PMCA-RE#351]
Do you have the same problem?  My camera started out with only Japanese, I unlocked the language using PMCA.  But when the last few languages ​​are selected, the camera suddenly freezes and restarts, probably around the Arabic language

**Comment by rogeriotoledo** [comment]
Can you change it to English?

**Comment by Lunaandi** [comment]
Yes.  I am a camera dealer.  Encountered the same situation.  Now I don't dare to provide services to customers anymore, I am very worried that the system will crash after updating the system in the future.  I've had a lot of A7R2s not booting at all in the past.
